examples/gossipsub-chat.rs

```diff
@@ -140,7 +138,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
                         swarm.behaviour_mut().gossipsub.remove_explicit_peer(&peer_id);
                     }
                 },
-                SwarmEvent::Behaviour(MyBehaviourEvent::Gossipsub(GossipsubEvent::Message {
+                SwarmEvent::Behaviour(MyBehaviourEvent::Gossipsub(gossipsub::Event::ProtobufMessage {
```

Comment:
Should it be `ProtobufMesssage` or maybe revert it to `Message`? I think that externally developer shouldn't care if the message is protobuf or other type, but not sure

Reply:
We can't easily deprecate enum variant names so I'd say revert it to `Message` so we remain non-breaking!

Reply:
done :heavy_check_mark: